conan/tools/files/files.py

```diff
@@ -117,7 +117,7 @@ def get(conanfile, url, md5=None, sha1=None, sha256=None, destination=".", filen
     :param auth:  forwarded to ``tools.file.download()``.
     :param headers:  forwarded to ``tools.file.download()``.
     :param strip_root: forwarded to ``tools.file.unzip()``.
-    :param extract_filter: forwarded to ``tools.file.unzip()``.
+    :param excludes: forwarded to ``tools.file.unzip()``.
```

Comment:
Not sure I understand, but why extract_filter was replaced here, even though it's still in the parameter list as well as in other places?

Reply:
I think it was a misunderstanding 😅. The param is still there (https://github.com/conan-io/conan/pull/18831/files#diff-9afbf22c7cefd3f7b642e9b1b988ba829d3c36e3092bfb6140d01ad0bba4c571R97), it's only missing the param in the docstring:

```suggestion
    :param extract_filter: forwarded to ``tools.file.unzip()``.
    :param excludes: forwarded to ``tools.file.unzip()``.
```

Reply:
Added back in https://github.com/conan-io/conan/pull/18850